src/CommandLineUtils/HelpText/DefaultHelpTextGenerator.cs

```diff
@@ -328,5 +345,21 @@ protected virtual string Format(CommandOption option)
             }
         }
 
+        string[] ExtractamesFromEnum(Type type)
+        {
+#if NETSTANDARD1_6 // no support for GetGenericArguments()
```

Comment:
Try using `type.GetTypeInfo().GetGenericArguments()`

https://docs.microsoft.com/en-us/dotnet/api/system.reflection.typeinfo.getgenericarguments?view=netstandard-1.6#System_Reflection_TypeInfo_GetGenericArguments

Reply:
Ill try but t.b.h. i don't think supporting 1.6 is worth it. It's a premature release with so many limitations that people with good reason are abandoning it

Reply:
I think we ahould add a test for attribute usage.. what is the best method for bootstrapping the code in unit tests